cat/CaT-service.yaml

```diff
@@ -4998,6 +4999,7 @@ components:
         - Alternative
         - MultiAnswerGroup
         - PercentageGroup
+        - PercentageOf
```

Comment:
This is fine but does the dependency value need adding in as an optional value. I think this is in for conditional dependency but not for relationships